Some powerpc systems (biou) don't have R_PPC_NUM defined in 32-bit
mode (why?).  So, #ifdef the case for R_PPC_NUM.


diff -Naurb symtabAPI.orig/src/dyninst/symtabAPI/src/relocationEntry-elf-ppc32.C symtabAPI/src/dyninst/symtabAPI/src/relocationEntry-elf-ppc32.C
--- symtabAPI.orig/src/dyninst/symtabAPI/src/relocationEntry-elf-ppc32.C	2011-04-07 14:52:23.000000000 -0500
+++ symtabAPI/src/dyninst/symtabAPI/src/relocationEntry-elf-ppc32.C	2012-09-21 15:01:21.000000000 -0500
@@ -102,7 +102,9 @@
         CASE_RETURN_STR(R_PPC_GOT_DTPREL16_LO);
         CASE_RETURN_STR(R_PPC_GOT_DTPREL16_HI);
         CASE_RETURN_STR(R_PPC_GOT_DTPREL16_HA);
+#ifdef R_PPC_NUM
         CASE_RETURN_STR(R_PPC_NUM);
+#endif
         CASE_RETURN_STR(R_PPC_EMB_NADDR32);
         CASE_RETURN_STR(R_PPC_EMB_NADDR16);
         CASE_RETURN_STR(R_PPC_EMB_NADDR16_LO);
